airbyte-webapp/src/services/connectorBuilder/ConnectorBuilderStateService.tsx

```diff
@@ -100,6 +95,11 @@ export const ConnectorBuilderFormStateProvider: React.FC<React.PropsWithChildren
   const manifestRef = useRef(derivedJsonManifest);
   manifestRef.current = derivedJsonManifest;
 
+  const [showLandingPage, setShowLandingPage] = useState(
```

Comment:
nit: `isEqual` isn't super cheap, would be nice to wrap it into an initializer function

Reply:
Moved this into the landing page component since we now only need to do this once on page load. I put it in a useEffect which should only run a single time since I'm using refs for the local storage values